canonical-web-and-design/charmhub.io
# Integration page

## Done

Added integrate tab and integration charts

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- Click on an operator and go to <operator_name>/integrate
- Check that there is a list of operators
- Go to /activemq/integrate and click on an operator
- Check that there is an integration chart with one integration
- Go to /advanced-routing/integrate and click on an operator
- Check that there is an integration chart with two integrations
- Go to /ambassador/integrate and click on an operator
- Check that there is an integration chart with three integrations
- Go to /aodh/integrate and click on an operator
- Check that there is an integration chart with four integrations
- Go to /apache-flume-hdfs/integrate and click on an operator
- Check that there is an integration chart with five integrations
- Open any of the integration charts on mobile and check that it is a vertical chart with a carousel

**Note**: This is hard-coded data so there may be anomalies between charts


## Issue / Card

Fixes #432 

## Screenshots

![integration_chart_desktop_one](https://user-images.githubusercontent.com/501889/97690441-e5558080-1a94-11eb-9fcb-bb6ac47efd84.png)

![integration_chart_desktop_two](https://user-images.githubusercontent.com/501889/97690424-dff83600-1a94-11eb-897c-daf0691f610b.png)

![integration_chart_desktop_3](https://user-images.githubusercontent.com/501889/97695297-bf7faa00-1a9b-11eb-9e58-5e254a4afecf.png)

![integration_chart_desktop_four](https://user-images.githubusercontent.com/501889/97690465-ec7c8e80-1a94-11eb-8d23-9874600f95b7.png)

![integration_chart_desktop_5](https://user-images.githubusercontent.com/501889/97695324-c9a1a880-1a9b-11eb-897a-27945747a94d.png)

![integration_chart_mobile](https://user-images.githubusercontent.com/501889/97690476-f0101580-1a94-11eb-8fc2-ab87c3383c36.png)
